test/unit/play-display-icon-test.js

```diff
@@ -25,84 +25,47 @@ describe('PlayDisplayIcon', function() {
             displayIcon = new PlayDisplayIcon(model, {}, element);
 
-            expect(displayIcon.icon.className).to.not.include('jw-ab-idle-label');
+            expect(displayIcon.icon.className).to.not.include('jw-idle-label');
             expect(displayIcon.icon.lastChild.className).to.not.include('jw-idle-icon-text');
-
         });
 
-        it('should not add class if config is invalid', function() {
-            model.set('idleButtonText', 'invalid text');
-
+        it('should not create element if displayPlaybackLabel is false', function () {
             displayIcon = new PlayDisplayIcon(model, {}, element);
 
-            expect(displayIcon.icon.className).to.not.include('jw-ab-idle-label');
+            expect(displayIcon.icon.className).to.not.include('jw-idle-label');
             expect(displayIcon.icon.lastChild.className).to.not.include('jw-idle-icon-text');
         });
 
-        it('should add class and text if config is "Watch Now"', function() {
-            model.set('idleButtonText', 'Watch Now');
-
-            displayIcon = new PlayDisplayIcon(model, {}, element);
-
-            expect(displayIcon.icon.className).to.include('jw-ab-idle-label');
-            expect(displayIcon.icon.lastChild.className).to.include('jw-idle-icon-text');
-            expect(displayIcon.icon.lastChild.textContent).to.equal('Watch Now');
-        });
-
-        it('should add class and text if config is "Click to Play"', function () {
-            model.set('idleButtonText', 'Click to Play');
-
+        it('should create element when displayPlaybackLabel is true', function () {
+            model.set('displayPlaybackLabel', true);
             displayIcon = new PlayDisplayIcon(model, {}, element);
 
-            expect(displayIcon.icon.className).to.include('jw-ab-idle-label');
+            expect(displayIcon.icon.className).to.include('jw-idle-label');
             expect(displayIcon.icon.lastChild.className).to.include('jw-idle-icon-text');
-            expect(displayIcon.icon.lastChild.textContent).to.equal('Click to Play');
+            expect(displayIcon.icon.textContent).to.equal(localization.playback);
         });
 
-        it('should add class and text if config is "Play"', function () {
-            model.set('idleButtonText', 'Play');
-
+        it('should set playback label to the value of localization.playback', function() {
+            model.set('displayPlaybackLabel', true);
+            localization.playback = 'Jugar';
             displayIcon = new PlayDisplayIcon(model, {}, element);
 
-            expect(displayIcon.icon.className).to.include('jw-ab-idle-label');
+            expect(displayIcon.icon.className).to.include('jw-idle-label');
             expect(displayIcon.icon.lastChild.className).to.include('jw-idle-icon-text');
-            expect(displayIcon.icon.lastChild.textContent).to.equal('Play');
+            expect(displayIcon.icon.textContent).to.not.equal('Play');
```

Comment:
not a big deal but it would be better to have a variable storing the default localization value instead of hard coding; that way if we change the default value again, we don't need to update this test
```
const defaultPlaybackLocalization = localization.playback;
localization.playback = 'Jugar';
...
expect(displayIcon.icon.textContent).to.not.equal(defaultPlaybackLocalization);
```